src/mainviews/create-job.tsx

```diff
@@ -121,12 +122,18 @@ export function CreateJob(props: ICreateJobProps): JSX.Element {
           envList[0].name
         )?.map(format => format.name);
 
-        props.handleModelChange({
+        const newModel = {
           ...props.model,
           environment: envList[0].name,
           computeType: newComputeType,
           outputFormats: outputFormats
-        });
+        };
+
+        if (envList[0].notifications_enabled && !props.model.notification) {
+          newModel.notification = { enableNotification: true };
+        }
```

Comment:
Similar to my previous comment about duplicate state, I believe this extra attribute can also be removed since we should be able to derive this from the other fields on the model when we need it. That way you won't have to worry about duplicate state getting out-of-sync.